### PR TITLE
Use verb-noun form of commands in make.ps1 instead of aliases

### PR DESCRIPTION
**Pull request by phrohdoh**

### CODE REVIEW

**Comment by ScottNZ** [comment]
Why?

**Comment by Phrohdoh** [comment]
No real reason other than following the Powershell best-practice of being verbose for others that may read/rewrite your code.

This is very simple and doesn't change behaviour, so if it isn't desired I'm not willing to argue for it.

**Comment by ScottNZ** [comment]
Fair enough. I'm pretty indifferent to this.

**Comment by RoosterDragon** [comment]
Needs a rebase.

**Comment by Phrohdoh** [comment]
Rebased.

File: make.ps1 [hunk 1]
```diff
@@ -184,7 +184,7 @@ elseif ($command -eq "check-scripts")
 {
 	if ((Get-Command "luac.exe" -ErrorAction SilentlyContinue) -ne $null)
 	{
-		echo "Testing Lua scripts..."
+		Write-Output "Testing Lua scripts..."
 		foreach ($script in ls "mods/*/maps/*/*.lua")
```
Comment (atlimit8) [inline make.ps1]: Shouldn't `ls` be `Get-ChildItem` then?

Comment (Phrohdoh) [inline make.ps1]: It could be, yes.

**Comment by penev92** [comment]
:+0.5: until I can test this, with @atlimit8's comment.

**Pull request closed without merging**